buildbot-conf/master.cfg

```diff
@@ -17,37 +17,75 @@
 # Absolute or relative path to buildbot per-project configureations.
 # For local Buildbot installation it is convenient to use the same directory
 # for main master.cfg and projects .conf files
-FP = "./"
+FP = './'
 
 c = BuildmasterConfig = {}
 
 ####### BUILDSLAVES
 
-# import the passwds
-
-# import private
-# reload(private)
+# import the passwords
+import private
+reload(private)
 
 # c['db_url'] = private.mysqlConnection
 
+c['db'] = {
+    'db_url' : "sqlite:///state.sqlite",
+}
+
 # the 'slaves' list defines the set of allowable buildslaves. Each element is
 # a tuple of bot-name and bot-password. These correspond to values given to
 # the buildslave's mktap invocation.
 from buildbot.buildslave import BuildSlave
 
-c['slaves'] = [
-    BuildSlave("cordova-ios-slave", "pass", max_builds=1),
-    BuildSlave("cordova-android-slave", "pass", max_builds=1),
-    BuildSlave("cordova-windows-slave", "pass", max_builds=1),
-    BuildSlave("cordova-win8-slave", "pass", max_builds=1),
-    BuildSlave("cordova-blackberry-slave", "pass", max_builds=1),
-    BuildSlave("cordova-common-slave", "pass", max_builds=3),
-]
+c['slaves'] = []
+
+c['slaves'].extend([
+
+    # used slaves
+    BuildSlave(
+        'cordova-ios-slave',
+        private.slave34Pwd,
```

Comment:
I believe we have our own `master.cfg` that is different from Apache build bot one. It serves an abstraction so that we will be able to make `cordova.conf` the same for Apache and for local usage.

We will not be able to track Apache `master.cfg` in this repo and we don't need to do it.

This way `master.cfg` may not include any info about Apache server and its slave names and their numbers in case it is confusing for developers not familiar with Apache build bot.